pkg/cfn/manager/nodegroup.go

```diff
@@ -0,0 +1,71 @@
+package manager
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/weaveworks/eksctl/pkg/cfn/builder"
+
+	"github.com/kubicorn/kubicorn/pkg/logger"
+)
+
+func (c *StackCollection) makeNodeGroupStackName(sequence int) string {
+	return fmt.Sprintf("eksctl-%s-nodegroup-%d", c.spec.ClusterName, sequence)
+}
+
+func (c *StackCollection) makeNodeGroupParams(sequence int) map[string]string {
+	return map[string]string{
+		builder.ParamClusterName:      c.spec.ClusterName,
+		builder.ParamClusterStackName: c.spec.ClusterStackName,
+		builder.ParamNodeGroupID:      fmt.Sprintf("%d", sequence),
+	}
+}
+
+func (c *StackCollection) CreateNodeGroup(errs chan error) error {
```

Comment:
There is still some duplication between here and `CreateCluster`, but I'd rather address that separately.